src/pages/admin/reports/components/funds-expenditures-section/components/common/edit-funds-expenditures-category-modal/EditFundsExpendituresCategoryModal.module.scss

```
@@ -158,21 +117,11 @@
-
-.error {
-    @include type.small-text-regular;
-    margin: 4px 0 0;
-    min-height: 20px;
-    color: c.$error;
-    visibility: visible;
-}
-
-.errorHidden {
-    visibility: hidden;
+    color: c.$bluish-black;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)